R/derive_param_clinbenefit.R

```diff
@@ -3,12 +3,22 @@
 #' Adds a parameter for clinical benefit/disease control
 #'
 #' @details
-#' Clinical benefit/disease control is first identified for looking for subjects
+#' Clinical benefit/disease control is first identified by looking for subjects
 #' having response status, and then derived for subjects that have at least one
 #' evaluable non-PD response assessment prior to first PD (Progressive Disease)
 #' (i.e., responses exclusive of `NA`, `NE`, `ND`, and `PD`) and after a specified
 #' amount of time from a reference date (`ref_start_window`).
 #'
+#' Note: the user can exclude any additional response values using the `filter_source`
+#' argument.
+#'
+#' Example: filter_source = PARAMCD == "OVR" & AVALC != "NON-CR/NON-PD"
```

Comment:
Note for future use: `filter_source = PARAMCD == "OVR" & AVALC != "NON-CR/NON-PD"` for code blocks